### DISCUSSION TITLE
Is it possible to use Deep Link to a Filter WebPart without Term Value?

### DISCUSSION THREAD

**Discussion by dontthroworanges** [microsoft-search/pnp-modern-search#3741] [Q&A]
Is there a way to exclude a Term's value and only use the Term Name when deep linking to a filter?

Such as:
https://mysite.sharepoint.com/sites/site/SitePages/DocumentSearch.aspx?f=[{"filterName":"RefinableString102","values":[{"name":"Comm Cabinet Photo","operator":0}]}]

Instead of:
https://mysite.sharepoint.com/sites/site/SitePages/DocumentSearch.aspx?f=[{"filterName":"RefinableString102","values":[{"name":"Comm Cabinet Photo",_"value":"\"ǂǂ436f6d6d20436162696e65742050686f746f\"_","operator":0,"disabled":false}],"operator":"and"}]

I have created a search page that searches a list for our various document types. I would like the user to be able to use this search, find the document type that they need, click an icon in the results web part and it take them to another page where the Search Results webpart is pre-filtered to that specific document type. 

I have had to make this additional Document Type search page since not all the Document Type Refiners show in the Filters Webpart on the Document Search page. Guessing there is a limit on how many refiners can be displayed... 

Anyway, what is the best way to achieve what I am looking for?

**Comment by kasperbolarsen** [comment]
Your approach seems ok to me.
I really doubt that you can skip the ID in the filters thing as we can have multiple terms using the same label/text value.
There is a hard limit on the number of refiner values returned by the API,  but it is in the hundreds,  how many do you get?

**Reply by dontthroworanges** [reply]
Since I'll have to use the term ID, what is the best way to make the relationship between the list item and the term? Just add the term to the item? And then if that is the case, how do I get the HEX term ID and pass it through the URL to the other page? 

Looks like there are exactly 100 "Document Type" refiners visible in the Filter webpart. Obviously I have many more terms than that ;).

**Reply by dontthroworanges** [reply]
I found a workaround for now. If I tell the Search Box to use Dynamic URL Fragments with ...aspx#RefinableString102:"Term Name" then the search results are filtered by the name alone.
